packages/components/psammead-navigation/README.md

```diff
@@ -0,0 +1,92 @@
+
+## Installation
+
+`npm install @bbc/psammead-navigation`
+
+## Props
+
+### Navigation
+
+<!-- prettier-ignore -->
+| Argument | Type | Required | Default | Example |
+| -------- | ---- | -------- | ------- | ------- |
+| children | node | Yes      | N/A     | `<NavigationUl><NavigationLi url="/" script={latin} active="true">Home</NavigationLi><NavigationLi url="/sport" script={latin}>{Sport}</NavigationLi></NavigationUl>` |
+
+### NavigationUl
+
+<!-- prettier-ignore -->
+| Argument | Type | Required | Default | Example |
+| -------- | ---- | -------- | ------- | ------- |
+| children | node | Yes      | N/A     | `<NavigationLi url="/" script={latin} active="true">Home</NavigationLi><NavigationLi url="/sport" script={latin}>{Sport}</NavigationLi>` |
+
+### NavigationLi
+
+<!-- prettier-ignore -->
+| Argument | Type    | Required | Default | Example  |
+| -------- | ------- | -------- | ------- | -------- |
+| url      | string  | Yes      | N/A     | `/sport` |
+| script   | object  | Yes      | N/A     |  `{ canon: { groupA: { fontSize: '28', lineHeight: '32',}, groupB: { fontSize: '32', lineHeight: '36', }, groupD: { fontSize: '44', lineHeight: '48', }, }, trafalgar: { groupA: { fontSize: '20', lineHeight: '24', }, groupB: { fontSize: '24', lineHeight: '28', }, groupD: { fontSize: '32', lineHeight: '36', }, }, }` |
+| dir      | string  | No       | `ltr`   | `rtl`    |
+| active   | boolean | No       | `false` | `true`   |
+
+## Usage
+
+```jsx
+import React from 'react';
+import Navigation, {
+  NavigationUl,
+  NavigationLi,
+} from '@bbc/psammead-navigation';
+import { latin } from '@bbc/gel-foundations/scripts';
+
+<Navigation>
+  <NavigationUl>
+    <NavigationLi url="/" script={latin} active="true">
+      Home
+    </NavigationLi>
+    <NavigationLi url="/sport" script={latin}>
+      {Sport}
+    </NavigationLi>
+    <NavigationLi url="/weather" script={latin}>
+      {Weather}
+    </NavigationLi>
+  </NavigationUl>
+</Navigation>;
+```
+
+### When to use this component
+
+The `Navigation` is designed to show a navigation bar on `index` pages, which will show all sections on a site. If there are too many items to fit on one line, the items will wrap to the next lines.
+
+### Accessibility notes
+
+The Navigation has a [`navigation` landmark](https://www.w3.org/TR/wai-aria-practices/examples/landmarks/navigation.html) to provide a way to identify links that are intended to be used for navigation.
+
+It includes a ["skip link"](https://www.w3.org/TR/WCAG20-TECHS/G1.html) giving users the option to skip to the main content before the assistive technology reads the full content of the interjection.
+
+We have added the role `list` and `listitem` to the `NavigationUl` and `NavigationList` respectively, due to a VoiceOver bug to reinstate the list semantics.
+
+We have also added a visually hidden text to let the user know which item in the Navigation is the current page. Note use of `role="text"` to stop splitting in VoiceOver.
```

Comment:
Can we update this to: "We have also added visually hidden text to let the user know which item in the Navigation is the current page. Note the use of visually hidden text here is due to lack of support at this time for the aria-current page attribute. Also note the use of `role="text"` to stop splitting in VoiceOver."